telegraf/telegraf.once.conf

```diff
@@ -0,0 +1,51 @@
+# A telegraf configuration to push metrics to InfluxDB 2.0 from
```

Comment:
I've created this conf, which is nice for testing and uploads just the metrics in the file with once option.